docs/content/user-guides/crd-acme/04-ingressroutes.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: simpleingressroute
+spec:
+  entrypoints:
+    - web
+  routes:
+  - match: Host(`your.domain.name`) && PathPrefix(`/notls`)
+    kind: Rule
+    services:
+    - name: whoami
+      port: 80
+
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: ingressroutetls
+spec:
+  entrypoints:
+    - websecure
+  routes:
+  - match: Host(`your.domain.name`) && PathPrefix(`/tls`)
+    kind: Rule
+    services:
+    - name: whoami
+      port: 80
+  tls:
```

Comment:
Is this required? It seems wrong that an "empty" tls section is required.

Reply:
you already found out in the meantime, but as a reference for others: see https://github.com/containous/traefik/pull/4654